tools/testbench/makefile-xcc

```diff
@@ -0,0 +1,127 @@
+# Build xt-run version of testbench
```

Comment:
Need to use cmake.

Reply:
This version builds with cmake. Though there's still error in install, but the xt executable runs OK.